ts/Extensions/ExportData.ts

```diff
@@ -1259,7 +1259,9 @@ Chart.prototype.hideData = function (): void {
 
 Chart.prototype.toggleDataTable = function (): void {
     var exportDivElements = this.exportDivElements,
-        menuItems = exportingOptions?.buttons?.contextButton.menuItems,
+        menuItems = this.options.exporting &&
+            this.options.exporting.buttons &&
+            this.options.exporting.buttons.contextButton.menuItems,
```

Comment:
I'm just curious, why `exporting.buttons` must be checked, but `exporting.buttons.contextButton` is always available? Any ideas @karolkolodziej and @bre1470 ?

Reply:
Because the exporting module might not be loaded.

Reply:
Though you are right. It is strange that contextButton is not tested. User could replace buttons with their own options and it would break.

Reply:
Hm.. I might be still missing something, sorry :) 
- `this.options.exporting` -> can be undefined, when exporting module not loaded
- `this.options.exporting.buttons` -> can be undefined because..?
- `this.options.exporting.buttons.contextButton` -> is always defined because exporting module was loaded

Reply:
I guess it can be undefined because it's an option?
```
interface ExportingOptions {
            allowHTML?: boolean;
            buttons?: ExportingButtonsOptions;
           ...
}
```

Reply:
I would say so too. You could set `buttons` to `undefined`, but the same is true for `contextButton`. So I guess that `contextButton` is an internal type bug.

BTW Maybe consider something like `const options = this.options.exporting` to make the `&&` condition a bit shorter.